public/locales/en/user.json

```diff
@@ -25,5 +25,10 @@
   "forms.errors.invalidCustomUrl.numbers": "Custom URL can't only contain numbers",
   "forms.errors.invalidCustomUrl.strangeCharacter": "Custom URL can't contain special characters",
   "forms.errors.invalidCustomUrl.duplicate": "Someone is already using this custom URL",
-  "forms.errors.invalidSens": "Motion sens can't be set if R-stick sens isn't"
+  "forms.errors.invalidSens": "Motion sens can't be set if R-stick sens isn't",
+
+  "auth.errors.aborted": "Login Aborted",
+  "auth.errors.failed": "Login Failed",
+  "auth.errors.discordPermissions": "For your sendou.ink profile, the site needs access to your Discord profile's name, avatar and social connections.",
+  "auth.errors.unknown": "The login via Discord failed for an unknown reason. If this keeps happening, please reach out for help."
```

Comment:
I realised that these keys have to be part of the common translations as they can be shown on any page the way it is currently set up, and we don't really want to load the `user` translations for all pages.

I didn't get how remix-i18next worked until just now 😅

Reply:
Ah! Very true. Yeah I guess you got it but we load what translations each page has on the handle function :)

```
export const handle = {
  i18n: "calendar",
};
```